projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1549,8 +1551,10 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * @hidden
      * @internal
      */
-    public handleOpening(event: CancelableEventArgs) {
-        this.onOpening.emit(event);
+    public handleOpening(event: CancelableBrowserEventArgs & IBaseEventArgs) {
+        const eventArgs: CancelableBrowserEventArgs & IBaseEventArgs = Object.assign({}, event, { owner: this });
+        this.onOpening.emit(eventArgs);
+        event.cancel = eventArgs.cancel;
         if (event.cancel) {
```

Comment:
remove the `if` statement please. It really do nothing here :)

Reply:
Not sure why that's left there, didn't pay it any attention - thanks for pointing it out!